appveyor.yml

```diff
@@ -7,7 +7,7 @@ clone_folder: c:\projects\src\github.com\koding\kite
 environment:
  PATH: c:\projects\bin;%PATH%
  GOPATH: c:\projects
- GOVERSION: 1.8
```

Comment:
might also be 1.9.2

Reply:
updated to 1.9.1 since we use 1.9.1 on koding/koding too